cabal.project

```diff
@@ -230,6 +233,8 @@ source-repository-package
     libs/cardano-ledger-core
     libs/cardano-ledger-pretty
     libs/cardano-protocol-tpraos
+    libs/compact-map
+    libs/ledger-state
```

Comment:
You shouldn't need to add this package, it is only used internally by ledger for memory benchmarking:
```suggestion
```

Reply:
Removed.  Thanks!